medusa/show/recommendations/recommended.py

```diff
@@ -241,9 +241,10 @@ def update_recommended_series_cache_index(indexer, new_index):
     :param indexer: Indexer in the form of a string. For example: 'imdb', 'trakt', 'anidb'.
     :new_index: Iterable with series id's.
     """
-    index = recommended_series_cache.get(binary_type(indexer)) or set()
-    index.update(set(new_index))
-    recommended_series_cache.set(binary_type(indexer), index)
+    if new_index:
```

Comment:
@p0psicles 
I don't mind keep it in, but why did you make this change in your PR?
It doesn't seem to be related to postProcessing in any way.

Reply:
Aaah.. It was a bug with the anidb recommended shows. Just tested that. Ill remove it here.